virttest/utils_test/__init__.py

```diff
@@ -433,6 +433,12 @@ def find_python(session, try_binaries=('python3', 'python2', 'python')):
         if session.cmd_status("which %s" % python) == 0:
             return python
 
+def find_host_python(try_binaries=('python3', 'python2', 'python')):
+    for python in try_binaries:
+        out = process.run("which %s" % python, shell=True)
+        if out.exit_status == 0:
+            return python
+
```

Comment:
Even use this method,  we can improve `find_python` to make it can work with both host and guest, according to the value of `session` .
```
if session:
    s = session.cmd_status(***)
else:
    s = process.run(***).exist_status
```

Reply:
How about simply using `session.cmd_status("%s --version" % python)`, that should work for linux as well as windows :wink:

Reply:
Great, it is available for both linux and windows :)